hack/debug-network.sh

```diff
@@ -0,0 +1,729 @@
+#!/bin/bash
+
+# This script will find any network connectivity issues with OpenShift SDN.
+# It performs: pod<->pod, pod<->node, pod<->service, pod<->external checks.
+# Assumes: SSH access to openshift master and openshift nodes.
+
+# die() echoes args to stderr and exits
+function die() {
```

Comment:
Is there a need for this to be in one file? I still think it would be much better to break these libraries out -- even if we have the goal of being standalone and not mingling them with the other Bash in this repo.